HealthMonitoring.AcceptanceTests/Helpers/Credentials.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace HealthMonitoring.AcceptanceTests.Helpers
+{
+    class Credentials
+    {
+        public Guid Id { get; set; }
```

Comment:
I think **Id** is more applicable to endpoint than username (in our context). Or not?